app/src/main/java/com/example/ithaca_transit_android_v2/states/SearchState.kt

```diff
@@ -0,0 +1,64 @@
+package com.example.ithaca_transit_android_v2.states
+
+import com.example.ithaca_transit_android_v2.models.Coordinate
+import com.example.ithaca_transit_android_v2.models.Location
+
+// Parent Class
+sealed class SearchState(
+    val startLocation: Location = Location("", Coordinate(0.0, 0.0), ""),
+    val endLocation: Location = Location("", Coordinate(0.0, 0.0), "")
+) {}
+
+// Default UI when launching app - only a search bar (no interactions so far)
+class LaunchState() : SearchState()
+
+// First time launching app then clicking on the search bar, no favorite or recent destinations
+class EmptyInitSearchState() : SearchState()
+
+/*
+ *Subsequent launches of app the clicking on search bar -> Displays Favorite and Recent
+ *Destinations. **Will be different from EmptyInitSearchState after v.0**
```

Comment:
you need spaces after all of the `*`, same thing as the `//` comments, so this should be:

```
/*
 * Subsequent ...
 * Destinations ...
 */
```